book-domain-service/app/routes.py

```diff
@@ -1,14 +1,30 @@
-from app import app
-from flask import request, Response
-from app.book import Book
+from flask import request, Response, jsonify
+from flask import current_app
```

Comment:
All of the imports from flask could be combined into a single import statement. If the line gets too long you can import as a tuple too:

```
from flask import (
    Blueprint,
    current_app,
    jsonify,
    Response,
    request,
)
```